src/EntityListener/VoteListener.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace AppBundle\EntityListener;
+
+use AppBundle\Entity\IdeasWorkshop\Vote;
+use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
+
+class VoteListener
+{
+    public function prePersist(Vote $vote, LifecycleEventArgs $args)
+    {
+        $vote->getIdea()->incrementVotesCount(1);
```

Comment:
The default parameter value is not needed

Reply:
Done.